noormadkour/howdy-neighbor
# Seeded Database with Fixtures, and added Comments View

# CommentsViewSet now up and running, and database is Seeded

**New Supported Routes**
1. `/comments`

**Models**
--no new models--

## STEPS TO TEST

1. Pull down the `mm-viewsandfixtures` branch and switch to it
2. Start/Restart your debugger/JSON-Server
3. Open Postman
4. GET Request `http://localhost:8000/` and verify you see all the available routes. You should see a new route of : -
- `http://localhost:8000/comments` 
5. GET request the above new route, and you should get all comments in the DB with `200 OK` status code

NOTE: Each comment should have all its info expanded via serializer, and the is_owner properties should function as expected
